runtimes/common/src/migrations.rs

```diff
@@ -58,12 +71,39 @@ where
 	}
 
 	#[cfg(feature = "try-runtime")]
-	fn post_upgrade(_state: sp_std::vec::Vec<u8>) -> Result<(), &'static str> {
+	fn post_upgrade(_state: sp_std::vec::Vec<u8>) -> Result<(), TryRuntimeError> {
+		use frame_support::ensure;
+
 		log::info!("RemoveInsecureRandomnessPallet::post_upgrade() checks 🔍");
-		if frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b"") {
-			Err("Storage in pallet_insecure_randomness_collective_flip is not empty after migration.")
-		} else {
-			Ok(())
-		}
+
+		ensure!(
+			!frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b""),
+			TryRuntimeError::Other(
```

Comment:
Also here.

Reply:
updated in: [aad9105](https://github.com/KILTprotocol/kilt-node/pull/541/commits/aad9105841f0aaa122c447a7832bc022739cff10)